### ISSUE TITLE
"Verify this session" dialogues don't appear in "other" sessions

### ISSUE THREAD

**Issue by JOJ0** [element-hq/element-web#13938]
### Description

I just upgraded my Riot-Web Installation from 1.5.15 to 1.6.4

Cross-Signing does not seem to work

### Steps to reproduce

- One session has been verified using the "Recovery Passphrase" already, running in browser **Safari**
- Log into Riot with new browser (eg. **Chrome**)
- "Verify this session" screen appears and waits that users connect to "other" compatible session (1.6.0+ or latest iOS in my case)
- Go to any other compatible riot-web session or riot-iOS session (eg: **Safari** session or **iOS session**)
- No qr-code screens and/or emoji-screens appear on the other sessions. The verification process appears not to be working
- Only the last resort solution "Use Recovery Passphrase" seems to be working.

I would expect that when I head over to another session, automatically the "verification qr-code, emoji screens appear and walk me through the cross-signing process"

Logs being sent: yes

![Screen Shot 2020-06-05 at 19 16 26](https://user-images.githubusercontent.com/2733783/83907984-9bc36c80-a766-11ea-9a59-ce47b1a4b8a9.png)

![Screen Shot 2020-06-05 at 19 16 32](https://user-images.githubusercontent.com/2733783/83908003-a5e56b00-a766-11ea-930d-80b0fb4e87e2.png)

### Version information

- **Platform**: web (in-browser)

For the web app:

- **Browsers**:
    - Verified session: Safari 13.1 (13609.1.20.111.8)
    - Unverified/to be verified session: Chrome: 83.0.4103.61 (Official Build) (64-bit)
- **OS**: macOS
- riot-web self-hosted version 1.6.4

iOS App:
- version 0.11.5 (latest as of 2020-06-05)

**Comment by t3chguy** [comment]
![image](https://user-images.githubusercontent.com/2403652/83917343-979b4d00-a76e-11ea-8a92-4961e16350b2.png)

Where is this screenshot from?

---

When I log in with a new session `B`, I observe the following:

![image](https://user-images.githubusercontent.com/2403652/83917468-d9c48e80-a76e-11ea-80da-10356363eb3f.png)

At the same time, on session `A` I observe:

![image](https://user-images.githubusercontent.com/2403652/83917485-e1843300-a76e-11ea-8c71-7f0917a32344.png)

upon clicking `Verify` on `A` I get the following on A:

![image](https://user-images.githubusercontent.com/2403652/83917499-eb0d9b00-a76e-11ea-9cfe-d7cb632a85fd.png)

Upon clicking `Continue` I get the following on A and B respectively:

![image](https://user-images.githubusercontent.com/2403652/83917520-f791f380-a76e-11ea-8b98-2be24c624f76.png)
![image](https://user-images.githubusercontent.com/2403652/83917525-f9f44d80-a76e-11ea-98b9-edf35963b1e7.png)

**Comment by Ghabry** [comment]
This also happened to me once. On one device I got no prompt on any other device about cross-signing/device trust. Had to use the recovery passphrase but on all others it worked. So this issue seems to be quite random?

**Comment by JOJ0** [comment]
> ![image](https://user-images.githubusercontent.com/2403652/83917343-979b4d00-a76e-11ea-8a92-4961e16350b2.png)
> 
> Where is this screenshot from?

When I log in to the "new" session in Chrome browser. It pops up in the very top-left of the browser window.


> When I log in with a new session `B`, I observe the following:

Thanks for your pictures, this clarifies what to expect, In my case just nothing happens when opening any of my other sessions.

**Comment by t3chguy** [comment]
![image](https://user-images.githubusercontent.com/2403652/83950920-0b4c6100-a826-11ea-9f95-5de88064e990.png)

So both of those screenshots were from the same session and in the backwards order?

**Comment by JOJ0** [comment]
same session in the order I posted them.

**Comment by Chaformbintrano** [comment]
The cross-verification crap worked only ONCE in my Riots. I have no idea how the authors could force the users to use such an atrocity.

**Comment by jryans** [comment]
@Chaformbintrano Clearly you are frustrated, but anger is not going to achieve anything. Please create a new issue for whatever you've experience, as otherwise we don't know about it, and we can't fix what we don't know about.

**Comment by JOJ0** [comment]
Hi,
if you need anythin else please get back to me. I sent logs, did you actually receive them?
Thanks a lot for your time and all the best!
Jojo

**Comment by jryans** [comment]
Yes, we received the debug logs you included when originally filing. Thanks, we'll take a look.

**Comment by Ghabry** [comment]
Is kinda unrelated I guess but you could also try this:
https://github.com/vector-im/riot-web/issues/13824#issuecomment-651824685

**Comment by richvdh** [comment]
This seems to have bitrotted. If folks can still reproduce this, please say so. Otherwise I'm closing.
